webapp/components/admin_console/ldap_settings.jsx

```diff
@@ -89,7 +89,7 @@ export default class LdapSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.ldap.enableTitle'
-                            defaultMessage='Enable Login With LDAP:'
+                            defaultMessage='Enable login with LDAP:'
```

Comment:
Propose we use "sign-in" to match the UI, rather than "login" which isn't used in the UI?